### ISSUE TITLE
Repo not set correctly on CentOS Stream 9

### ISSUE THREAD

**Issue by ncsutmf** [indiana-university/puppet-duo_unix#30]
There is no CentOS 9, only CentOS Stream.  The repos for Stream are at a different path.  It looks like there is a simple fix in the `duo_unix:::repos` class [here](https://github.com/indiana-university/puppet-duo_unix/blob/master/manifests/repo.pp#L71-L74) by changing it to:

```puppet
    $osname = $facts['os']['name'] ? {
        'CentOS'       => $facts['os']['release']['id'],
        default        => 'RedHat',
      }
```

More Info on the os release id fact and CentOS Stream:  https://github.com/puppetlabs/facter/pull/2291#issuecomment-790406132

This would also have the effect of pointing CentOS 8 Stream at the CentOSStream repo instead of the CentOS repo path.

Also noticed CentOS Stream "support" was added but then removed in https://github.com/indiana-university/puppet-duo_unix/commit/6c009d869b86a26a4876fa2af18d249ff8c359ca due to the os name fact still being set to CentOS.   Most of that can probably be added back with some additional changes to the repo spec test.

I can probably submit a Pull Request but it will take me a bit to set up signing of commits.

**Comment by ncsutmf** [comment]
Ended up setting up commit signing faster than expected and when ahead and created a Pull Request.

**Comment by avitacco** [comment]
Fixed with #31
